HACKING.md

```diff
@@ -92,6 +92,20 @@ oc annotate secret/resultsdb-auth  \
     jenkins.io/credentials-description="ResultsDB authentication"
 ```
 
+### Create Fedora Matrix notifications secret
+
```

Comment:
Can we mention here that it's available in BitWarden and specify its name there?